### PR TITLE
feature/CVSB-7943 - VTM Core Landing Page

### PR DESCRIPTION
**Pull request by irina005**
https://jira.dvsacloud.uk/browse/CVSB-7943

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
 import {AuthenticationGuard} from 'microsoft-adal-angular6';
+import {LandingPageComponent} from '@app/landing-page/landing-page.component';
+import {TechnicalRecordCreateComponent} from '@app/technical-record-create/technical-record-create.component';
 
 const routes: Routes = [
   {path: '', redirectTo: 'home', pathMatch: 'full', canActivate: [AuthenticationGuard]},
-  {path: 'home', loadChildren: './shell/shell.page.module#ShellPageModule' , canActivate: [AuthenticationGuard] },
+  {path: 'home', pathMatch: 'full', component: LandingPageComponent , canActivate: [AuthenticationGuard]},
+  {
+    path: 'home/search',
```
Comment (rspasici) [inline src/app/app-routing.module.ts]: `/home`, `/search` and `/create` shall all be independent routes like the technical-record one

File: src/app/technical-record-search/technical-record-search.module.ts [hunk 2]
```diff
@@ -27,11 +26,7 @@ import { VehicleTechRecordModelEffects } from '@app/store/effects/VehicleTechRec
     HttpClientModule,
     RouterModule.forChild([
       {path: '', component: TechnicalRecordSearchComponent, canActivate: [AuthenticationGuard]},
-      {
```
Comment (m-ujaffer) [inline src/app/technical-record-search/technical-record-search.module.ts]: add routes in a separate file. Follow approach similar to the following link: https://angular.io/guide/lazy-loading-ngmodules

Comment (m-ujaffer) [reply src/app/technical-record-search/technical-record-search.module.ts]: We would need the routes once you move "create" component over here.

Comment (m-ujaffer) [reply src/app/technical-record-search/technical-record-search.module.ts]: It should be similar to:

```
import { Routes, RouterModule } from '@angular/router';

import { TechnicalRecordComponent } from './PATH_TO_COMPONENT';
import { TechnicalRecordCreateComponent } from './PATH_TO_COMPONENT';


const routes: Routes = [
  {
    path: '',
    component: TechnicalRecordComponent
  },
   {
     path: 'create',
    component: TechnicalRecordCreateComponent
  }
];
export class TechnicalRecordRoutingModule { }
```

**Comment by m-ujaffer** [comment]
After discussion with @irina005 this component will start journey for the vehicle creation and will only take initial input i.e. vehicle type and vin/vrm.

**Pull request merged**